### PR TITLE
Update to Chromium 79.0.3945.79

### PR DESCRIPTION
**Pull request by tangalbert919**
Chromium 79 was released to the stable branch today.

It might be best to do a squash merge on this.

### CODE REVIEW

**Comment by jstkdng** [comment]
does this release contain the changes related to ad blockers?

**Comment by tangalbert919** [comment]
If you are talking about Google's Manifest V3, then I don't think so. According to Google's [Migrating to Manifest V3](https://developer.chrome.com/extensions/migrating_to_manifest_v3) document, it's mostly in preview, so Chrome 80 or 81 might have those changes (both of which are in the dev/canary branches).

**Comment by Zoraver** [comment]
@tangalbert919 have you successfully compiled Chromium with this patchset?

**Comment by tangalbert919** [comment]
Yes, I have.

File: patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch [hunk 1]
```diff
@@ -1,8 +1,18 @@
 # Additional changes to Inox's fix-building-without-safebrowsing.patch
 
+--- a/chrome/browser/browser_resources.grd
++++ b/chrome/browser/browser_resources.grd
+@@ -597,7 +597,6 @@
+       </if>
+       <if expr="not is_android">
+         <include name="IDR_TAB_RANKER_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="resource_coordinator\tab_ranker\example_preprocessor_config.pb" type="BINDATA" />
+-        <include name="IDR_TAB_RANKER_PAIRWISE_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="resource_coordinator\tab_ranker\pairwise_preprocessor_config.pb" type="BINDATA" />
```
Comment (Eloston) [inline patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch]: We should add an exception for this file to `devutils/update_lists.py` next to `example_preprocessor_config.pb`

File: patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch [hunk 2]
```diff
@@ -257,22 +186,185 @@
...
+                                    const GURL& frame_url) override;
+   safe_browsing::PasswordProtectionService* GetPasswordProtectionService()
+--- a/chrome/browser/resource_coordinator/tab_ranker/tab_score_predictor.cc
++++ b/chrome/browser/resource_coordinator/tab_ranker/tab_score_predictor.cc
+@@ -310,17 +310,6 @@ void TabScorePredictor::LazyInitialize()
+     DCHECK_EQ(preprocessor_config_->feature_indices().size(),
+               static_cast<std::size_t>(tfnative_model::FEATURES_SIZE));
+   }
+-
+-  if (type_ == kPairwiseScorer) {
```
Comment (Eloston) [inline patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch]: We also need to revert this change as part of adding back the binary file exception.

**Comment by Eloston** [comment]
On second thought, I'll take care of this myself.

**Comment by jstkdng** [comment]
hey @Eloston , could you create a tag for this new version? thanks
